src/components/stepforms/UnpivotStepForm.vue

```diff
@@ -0,0 +1,129 @@
+<template>
+  <div>
+    <step-form-title :title="title"></step-form-title>
+    <WidgetMultiselect
+      id="keepColumnInput"
+      v-model="editedStep.keep"
+      name="Keep"
+      :options="columnNames"
+      @input="setSelectedColumns({ column: editedStep.keep[0] })"
+      placeholder="Add columns to keep"
+    ></WidgetMultiselect>
+    <WidgetMultiselect
+      id="unpivotColumnInput"
+      v-model="editedStep.unpivot"
+      name="Unpivot"
+      :options="columnNames"
+      @input="setSelectedColumns({ column: editedStep.unpivot[0] })"
```

Comment:
I'm not sure what we want here ? Its maybe when we focus a certain input field that we want to show selectedColumns in the table no ?

Reply:
@Dalanir Yes you're right, and more generally speaking I think we should review every interaction from every form. It may consist in a dedicated PR !  In the meantime, I see this as a kind of "default" behaviour.
Are you ok with that ?

Reply:
You're right, take care of this on specific PR, maybe we should open an issue about that ?

Reply:
Done !